internal/tools/aws/database_multitenant.go

```diff
@@ -206,36 +216,36 @@ func (d *RDSMultitenantDatabase) Provision(store model.InstallationDatabaseStore
 
 	err = d.createDatabaseIfNotExist(ctx, installationDatabaseName)
 	if err != nil {
-		return errors.Wrapf(err, "unable to create schema in multitenant RDS cluster ID %s", *lockedRDSCluster.cluster.DBClusterIdentifier)
+		return errors.Wrap(err, "failed to provision multitenant database because provisioner is unable to create a installation database")
```

Comment:
This is an example of one that really should have the error prefix wrapped somewhere up the stack. For example this should be something like `errors.Wrap(err, "failed to create new installation database").

Because this is the `Provision` method, the caller should take any errors it receives and wrap them with something like `errors.Wrap(err, "failed to provision database").

There are quite a few errors around here that could benefit from this type of improvement.